server/src/app.controller.ts

```diff
@@ -1,28 +1,57 @@
-import { Controller, Get } from '@nestjs/common';
-import { TestService, AppService } from './app.service';
+import { Controller, Get, Post, Body } from '@nestjs/common';
+import { TestService, BrowseService } from './app.service';
 import ApiFactory from './api/import';
 
 
 function getApi(description: string) {
   return ApiFactory.getInstance().getApi('v1.0', description);
 }
+
+function processRequest(func: Function, body): string {
+  return JSON.stringify(func(body));
+}
 @Controller()
 export class AppController {
   constructor(private readonly testService: TestService, 
-    private readonly appService: AppService) {}
+    private readonly browseService: BrowseService) {}
 
   @Get(ApiFactory.getInstance().getApi('test', '/test1/test2/get'))
   getTest(): string {
     return this.testService.getTest();
   }
 
   @Get(getApi('/browse/question-list/get'))
-  getBrowseQuestionList(): string {
-    return this.appService.getBrowseQuestionList();
+  getBrowseQuestionList(@Body() body): string {
```

Comment:
while returning a string works with the hardcoded values, it does not work when returning an actual DTO.

Also we aren't returning strings/HTML content, we're returning JSON - nest doesn't recognize strings as json this way

Reply:
This looks like a problem for you to solve. You should approve, check it in, then change from your branch?